src/transforms/groupby.js

```diff
@@ -39,11 +39,11 @@ exports.attributes = {
         valType: 'string',
         description: [
             'Pattern by which grouped traces are named. If only one trace is present,',
-            'defaults to the group name (`"%g"`), otherwise defaults to the group name',
-            'with trace name (`"%g (%t)"`). Available escape sequences are `%g`, which',
-            'inserts the group name, and `%t`, which inserts the trace name. If grouping',
+            'defaults to the group name (`"%{group}"`), otherwise defaults to the group name',
+            'with trace name (`"%{group} (%{trace})"`). Available escape sequences are `%{group}`, which',
```

Comment:
Yeah, you're right. There are two options here:
1. Expose special sequences. That's what I've done here. So only `group` and `trace` are available. They're custom.
2. Make this a more general pattern of evaluating trace properties, in which case it would indeed be `%{name}` and… well, there's not really a property for the group unless we add it custom which is why I went with 1 here.